### PR TITLE
new mention handler tests

### PR DESCRIPTION
**Pull request by botany-demo**
new mention handler tests

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -13,3 +13,5 @@ login page design updates
 fix auth redirect error
 
 new TOS page
+
+new mention handler tests
```
Comment (botany-lead) [inline readme.md:17]: this is cool @botany-demo

Comment (botany-lead) [reply readme.md:17]: check this out @botany-dummy-org-1/team_1

Comment (botany-lead) [reply readme.md:17]: seriously great @botany-demo, please look @botany-dummy-org-1/team_1

Comment (botany-demo) [reply readme.md:17]: thanks so much @botany-lead!

Comment (botany-demo) [reply readme.md:17]: thanks so much @botany-lead!

Comment (botany-demo) [reply readme.md:17]: and thank you @botany-dummy-org-1/team_1

Comment (botany-lead) [reply readme.md:17]: hey there @botany-demo

Comment (botany-lead) [reply readme.md:17]: hey there @botany-dummy-org-1/team_1

**Comment by botany-demo** [comment]
this is looking good @botany-dummy-org-1/team_1

**Pull request merged**